### ISSUE TITLE
Eigen 3.3.4 repository at bitbucket moved to gitlab

### ISSUE THREAD

**Issue by kjsanger** [wtsi-npg/npg_conda#174]
`ERROR:root:WARNING:conda_build.source:Error: HTTP 403 FORBIDDEN for url <https://bitbucket.org/eigen/eigen/get
/3.3.4.tar.bz2> `

https://bitbucket.org/eigen/eigen/get/3.3.4.tar.bz2

```
Repository unavailable

Bitbucket no longer supports Mercurial repositories.
```

**Comment by kjsanger** [comment]
Moved to https://gitlab.com/libeigen/eigen/-/archive/3.3.4/eigen-3.3.4.tar.bz2 in December 2019. That we didn't notice means we probably don't need the recipe any more!

**Comment by kjsanger** [comment]
Resolved by #188
